### DISCUSSION TITLE
Manual start the reading process through REST API

### DISCUSSION THREAD

**Discussion by pacman789** [jomjol/AI-on-the-edge-device#1051] [Ideas] [answered]
Hi,

First of all, thanks for your awesome project! I use it 1 year from now with 4 external LEDs and works perfectly. 
I think many of us are using GAS for heating system only.  I would like to take picture as often as possible when the boiler is on, rest of the time is more than enough if read only once in every 6 hours. So I set the interval to 360mins, but I would like to trigger the reading through REST API in every 3 mins when the boiler is on.
Is this feasible?

Or is there any risk  if i set the interval to 3 mins for 7x24?  

Thanks,
pacman789

**Comment by jomjol** [comment, marked as answer]
There is no risk in increasing the reading time to 3 minutes 24x7 - that's how mine is running since the beginning.

**Comment by whoami-pi** [comment]
Same question came in my mind, just to improve the energy consumption. I know its not so high but nowadays I‘m improving all the electricity consumptions. Trying to reduce wherever possible.

So, would be great having an option to trigger the picture via http.

Thanks, really great tool!!!!

**Reply by caco3** [reply]
i added a feature request for this: https://github.com/jomjol/AI-on-the-edge-device/issues/1351
